pkg/chartutil/create.go

```diff
@@ -96,6 +104,40 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+volumes: []
+  # - name: cache-volume
+  #   emptyDir: {}
+
+volumeMounts: []
+  # - name: cache-volume
+  #   mountPath: /cache
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 10001
+
+env: []
+  # - name: FOO
+  #   value: bar
+  # - name: BAZ
+  #   value: qux
+
+readinessProbe:
```

Comment:
I don't think a readinessProbe should be enabled by default. Not all applications respond to HTTP requests (in fact, Tiller's one such example), and not all of them respond with a 200 OK on the root URL.

Reply:
true I was just following what comes out of the box with helm create atm